source/fe/fe_nedelec_sz.cc

```diff
@@ -222,6 +222,67 @@ FE_NedelecSZ<dim, spacedim>::shape_grad_grad_component(
 
 
 
+template <>
+void
+FE_NedelecSZ<3, 3>::precompute_line_to_cell_map(
+  const DoFHandler<3, 3> &dof_handler)
+{
+  const int dim      = 3;
+  const int spacedim = 3;
+
+  // Compute the vertex to cell map
+  std::vector<
+    std::set<typename Triangulation<dim, spacedim>::active_cell_iterator>>
+    vertex_to_cell =
+      GridTools::vertex_to_cell_map(dof_handler.get_triangulation());
+
+  // Compute the line-neighbors
+  line_to_cell_map = std::vector<std::vector<
+    std::set<typename Triangulation<dim, spacedim>::active_cell_iterator>>>(
+    dof_handler.get_triangulation().n_active_cells(),
+    std::vector<
+      std::set<typename Triangulation<dim, spacedim>::active_cell_iterator>>(
+      GeometryInfo<dim>::lines_per_cell));
```

Comment:
Given this complicated setup, could `Table<2, std::set<typename Triangulation<dim, spacedim>::active_cell_iterator>` be a more appropriate data structure? It seems you only need the access via the pair  `active_cell_index / line_index_within_cell`, so a 2D array seems most efficient. (Keeping the efficiency of `std::set` aside, but that is not related to this PR.)